scripts/session.js

```diff
@@ -134,13 +135,34 @@ ws.onmessage = function(evt){
       this.className = 'hidden-point';
     });
     Cookies.remove('point');
+    $('#statistics').hide();
+    $('.point_count').remove();
   }
 
   // show all votes
   if(my_received_message.type == "show_all_votes"){
+    var pointArray = [];
     $("td[id]").each(function(){
       this.className = '';
+      if(this.innerText != '') {
+        pointArray.push(this.innerText);
+      }
+    });
+
+    var pointHash = {};
+    pointArray.forEach(function(point_value){
```

Comment:
We can move the logic about `pointHash` into `$("td[id]").each` directly, no need to create `pointArray`.